templates/nft-minting-dapp-template/frontend/hooks/useGetCollectionData.ts

```diff
@@ -79,15 +75,45 @@ async function getStartAndEndTime(collection_id: string): Promise<[start: Date,
   ];
 }
 
+async function getUserMintBalance(
```

Comment:
this should live in the `view-functions` folder https://github.com/aptos-labs/create-aptos-dapp/tree/main/templates/nft-minting-dapp-template/frontend/view-functions